File: src/snakegame/drawutils/AppColors.java
package snakegame.drawutils;

import java.awt.*;

/**
 * Created by Gilbert on 09-10-16.
 */
public final class AppColors {

    public static Color randomGrey() {
        int rValue = (int)(Math.random() * 50) + 50;
        return new Color(rValue, rValue, rValue);
    }

}

File: src/snakegame/drawutils/AppColors.kt
@file:JvmName("AppColors")

package snakegame.drawutils

import java.awt.Color

/**
 * Created by Gilbert on 09-10-16.
 */

fun randomGrey(): Color {
    val randomValue = (Math.random() * 50).toInt() + 50
    return Color(randomValue, randomValue, randomValue)
}
